server/src/main/java/chat/engine/Connect.java

```diff
@@ -0,0 +1,93 @@
+package chat.engine;
+
+import org.apache.log4j.Logger;
+
+import java.io.*;
+import java.net.Socket;
+import java.nio.charset.StandardCharsets;
+
+public class Connect implements Runnable {
+
+    private static final String CLIENT = "Client: ";
```

Comment:
you should not make constants for every word in sentence:) Move this to back to message